rust/gitxetcore/src/data/mdb.rs

```diff
@@ -335,14 +334,15 @@ pub async fn download_shards_to_cache(
     cache_dir: &Path,
     shards: Vec<MerkleHash>,
 ) -> errors::Result<Vec<PathBuf>> {
+    if shards.is_empty() {
+        return Ok(vec![]);
+    }
+
     let cas = create_cas_client(config).await?;
     let cas_ref = &cas;
 
-    let progress_reporter = DataProgressReporter::new(
-        &format!("{XET_PROGRAM_NAME}: Retrieving metadata"),
```

Comment:
This route is also used by pyxet, thus reporting `Git-Xet` is confusing and so change to `Xet`